docs/_scss/components/_accordions.scss

```diff
@@ -11,21 +11,6 @@ $accordion-border: 3px solid $color-gray-lightest;
     margin-top: 1rem;
```

Comment:
Fascinating.

Still needed the `$ bundle update` - just in case that shouldn't be necessary.

Can you point me to *why* install versus update makes the difference?

Reading the Terminal didn't seem much different.